locales/tr-TR/translation.json

```diff
@@ -0,0 +1,280 @@
+  "You have great ideas you want to share?": "Harika fikirleriniz mi var, paylaşmak ister misiniz ?",
+  "Tell us!": "Bize bahsedin!",
+  "New address": "Yeni adres",
+  "Refresh": "Yenile",
+  "Refresh data": "Veriyi yenile",
+  "Alephium Logo": "Alephium Logosu",
+  "Alephium": "Alephium",
+  "WALLET": "CÜZDAN",
+  "CURRENT WALLET": "ŞU ANKİ CÜZDAN",
+  "MENU": "MENÜ",
+  "Overview": "Genel görünüm",
+  "Lock": "Kilitle",
+  "Enter password": "Şifreyi gir",
+  "Login": "Giriş",
+  "Security Check": "Güvenlik doğrulaması",
+  "Alright! Time to check if you got your words right!": "Pekala! Şimdi kurtarma sözcüklerini doğru şekilde aldığınızı kontrol etme zamanı!",
+  "Select the words in the right order. Ready?": "Doğrulama sözcüklerini doğru şekilde sıralanması için seçin. Hazır mısınız ?",
+  "Ready!": "Hazır!",
+  "Something went wrong when creating encrypted wallet.": "Şifrelenmiş cüzdan oluşturulurken birşeyler yolunda gitmedi.",
+  "Select the words in the right order.": "Kurtarma sözcüklerini doğru sırayla seçin.",
+  "It seems like you made a mistake in the words' order. But don't worry, you can reorder the words by dragging them around.": "Kurtarma sözcüklerini yanlış sıralama yaptığınız görülüyor ama meraklanmayın. Sözcüklerin yerlerini doğru sıralamada tekrar değiştirip sıralayabilirsiniz.",
+  "Great job! Remember to keep those words safe.": "Harika! Kurtarma sözcüklerini güvenli bir yere sakladığınızı unutmayın",
+  "Continue": "Devam",
+  "Password is too weak": "Şifre çok zayıf",
+  "Insecure password": "Günensiz şifre",
+  "Wallet name is too short": "Cüzdan adı çok kısa",
+  "Wallet name already taken": "Cüzdan adı daha önceden alınmış",
+  "New wallet name": "Yeni cüzdan ismi",
+  "New password": "Yeni şifre",
+  "Retype password": "Şifreyi tekrar yazın",
+  "Passwords are different": "Şifre uyuşmazlığı",
+  "Make sure to keep your password secured as it cannot be changed in the future.": "Şifrenizi güvenli şekilde sakladığınızdan emin olunuz, daha sonrasında değiştirilemeyeceğini bilin.",
+  "Alephium doesn't have access to your wallet.\nYou are the only owner.": "Alephium projesinin herhangi bir şekilde cüzdanınıza erişme şansı yoktur. Cüzdanınız yalnızca sizin sahipliğinizdedir.",
+  "Back": "Geri",
+  "Type your recovery phrase": "Kurtarma sözcüklerinizi yazın",
+  "Error while importing wallet": "Cüzdan içe aktarılırken hata oluştu",
+  "Make sure to store the words in a secure location! They are your wallet's secret recovery phrase.": "Kurtarma sözcüklerini güvenli bir yere sakladığınızdan emin olun! Cüzdanınıza özel gizli kurtarma sözcükleridir.",
+  "All good? Let's continue!": "Herşey yolunda mı ? Haydi devam edelim!",
+  "Everything is ready!": "Herşey hazır!",
+  "Welcome to Alephium.": "Alephium' a hoşgeldiniz.",
+  "Let's go!": "Haydi başlayalım!",
+  "welcomeScreenOneAddressPerGroupMessage": "Gelişmiş kullanıcı: <1>her bir gruba madencilik ya da defi için bir adres</1> ile başlamak ister misiniz ?",
```

Comment:
"Gelişmiş kullanıcı: <1>her bir gruba madencilik ya da DeFi için bir adres</1> ile başlamak ister misiniz?",